test/functional/helpers.lua

```diff
@@ -441,6 +442,9 @@ local curtabmeths = create_callindex(curtab)
 
 return function(after_each)
   if after_each then
+    after_each(function()
+      os.remove('Xtest_rplugin_manifest')
```

Comment:
Do you mean just the `os.remove('Xtest_rplugin_manifest')` line?  Wouldn't that cause it to try and delete a file before it's created and never delete it after tests finish?  (If it's ever created, that is)